### DISCUSSION TITLE
Install Hydra in EKS POD

### DISCUSSION THREAD

**Discussion by Newbie112222** [facebookresearch/hydra#2685] [Q&A]
Hi All,
I have spun up a pod on my EKS cluster and i wanted some guidance for installing Hydra on this pod. This pod just has awscli and curl installed for now.

What i am trying to test is to connect this pod to a new RDS instance and run curl commands to fetch data from the instance. Any help would be appreciated

**Comment by Jasha10** [comment]
Hi @Newbie112222,

Does your pod have python installed? Does it have pip installed?
The most common way to install Hydra is via `pip`:
```
pip install hydra-core
```
